packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonicContainer.js

```diff
@@ -49,14 +49,11 @@ export default class ConfirmMnemonicContainer extends Component {
     }
   }
 
-  handleClickWord = (word) => {
-    if (!this.state.confirmPhrase.includes(word)) {
-      this.setState(
-        { confirmPhrase: this.state.confirmPhrase.concat(word) },
-        this.refreshCurrentMnemonicForm
-      )
-      return true
-    }
+  handleClickWord = (wordItem) => {
+    this.setState(
+      { confirmPhrase: this.state.confirmPhrase.concat(wordItem) },
+      this.refreshCurrentMnemonicForm
```

Comment:
Why we name it "refreshCURRENTmnemonicForm"? Is there any other Mnemonic form at the same time? It's better to remove current word from param name

Reply:
Done